db/python/tables/sample.py

```diff
@@ -97,6 +97,78 @@ async def update_sample(
         await self.connection.execute(_query, {**values, 'id': id_})
         return id_
 
+    async def merge_samples(
+        self,
+        id_keep: int = None,
+        id_merge: int = None,
+        author: str = None,
+    ):
+        """Merge two samples together"""
+        sid_merge = sample_id_format(id_merge)
+        _, sample_keep = await self.get_single_by_id(id_keep)
+        _, sample_merge = await self.get_single_by_id(id_merge)
```

Comment:
Can do these all in one to avoid multiple rounds to the database, using:

```suggestion
        samples =  await get_samples_by(sample_ids=[id_keep, id_merge])
        samples_map = {s.id: s for s in samples}
        sample_keep = samples_map[id_keep]
        sample_merge = samples_map[id_merge]
```